src/Clock.jsx

```diff
@@ -101,14 +101,9 @@ export default function Clock({
     if (newUI) {
       return (
         <>
-          {renderInnerRing()}
+          {renderClockFaceBackground()}
           {renderViviLogo()}
-          <div className="react-clock__face-outer" style={{ backgroundColor }}>
-            <div className="react-clock__face-inner">
-              {renderMinuteMarksFn()}
```

Comment:
What happened to `renderMinuteMarksFn`?

Reply:
Didn't notice this because we don't even actually use this. 

I've added it back in there leave it there for the sake of having the param available, but in Sample.tsx and our use of it in vivi-widgets, we set the renderMinuteMarks param to false.